### DISCUSSION TITLE
Release 1.14

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#11] [Announcements]
- Added Clear Chart button to Chart Settings
- Added MQTT client (Not Fully Tested with encrypted protocol and user login)
- Some UI Fixes for Multi Language
- Get Setting from old installation on first run

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.14'>Release 1.14</a>.</em>

**Comment by webspiderteam** [comment]
for testing output u can use https://github.com/webspiderteam/Bluetooth_DMM_For_Windows_MQTT_Test_Project

**Comment by ludwich66** [comment]
Now i'am back - Time for Testing ;-)

**Comment by webspiderteam** [comment]
welcome back :)  waiting for test results ;)
